tests/e2e/main.go

```diff
@@ -240,10 +240,70 @@ var rootCmd = &cobra.Command{
 		}
 
 		// create status CRD
-		if statusCRD, err := ioutil.ReadFile(filepath.Join(assetsPath, manifestsFolder, "status-crd.yaml")); err != nil {
+		if CRDBytes, err := ioutil.ReadFile(filepath.Join(assetsPath, manifestsFolder, "status-crd.yaml")); err != nil {
 			glog.Fatalf("Error reading %#v", err)
 		} else {
-			CRDObj, _, err := decode([]byte(statusCRD), nil, nil)
+			CRDObj, _, err := decode([]byte(CRDBytes), nil, nil)
```

Comment:
CRDBytes is already a byte array, no need for conversion here.

Reply:
this is converting bytes into a runtime.Object them cast to a specfic api object

Reply:
`ioutil.ReadFile` returns `[]byte` and decode accepts `[]byte` so no need for `[]byte(CRDBytes)`

Reply:
oh right, seems we are doing the same everywhere in integration. I will either polish or maybe just drop "integration" in a follow up since its value is minimum now we have aws-e2e-operator in place.
If there's no other concern I'd suggest to get this in so I can put a few follow ups in place blocked by this one

Reply:
sure sounds good